tests/pillar/system.sls

```diff
@@ -2,10 +2,11 @@ linux:
   system:
     enabled: true
     cluster: default
-    name: test01
+    name: linux
```

Comment:
we don't want to have 30 unit tests to spin 30 containers with CI. So we tend to have test/pillar for single/cluster/system/storage/network and possibly no more. I did this changes as I locally already test the formula with kitchen.